R/tax_certainty.R

```diff
@@ -0,0 +1,33 @@
+#' Check if a taxon is certain or uncertain.
+#'
+#' Check whether a given taxon in a vector is certain or uncertain.
+#' Binomial names (genus and species) are considered "certain",
+#' whereas names containing only the genus,
+#' or genus and not determined species is considered "uncertain".
+#'
+#' @param taxon_names Character vector containing the species names to be
+#' checked.
+#' It allows taxon names separated by underscore or space.
+#' @return Numeric vector classifying each species as certain (1) or uncertain
+#' (0).
+#' @examples
+#' sirenia_sp <- c("Metaxytherium indet.", "Felsinotherium_serresii",
+#'                  "Felsinotherium gervaisi", "Potamosiren indet.",
+#'                  "Potamosiren", "Metaxytherium sp.")
+#'
+#' certainty <- tax_certainty(sirenia_sp)
+#'
+#' @export
+tax_certainty <- function(taxon_names) {
```

Comment:
I'm going to veto this suggested change (not necessarily the comment) because by not assigning a value here, R automatically throws a useful error if an argument isn't provided.

Reply:
That's true, but the current error handling is checking for a NULL input argument anyway which gives a more explicit message of what the input should be. That said, I am not going to lose any sleep over this being implemented or not - I can't imagine many users will forget to specify an input.